api/models/Machine.js

```diff
@@ -26,6 +26,7 @@ const url = require('url');
 const Promise = require('bluebird');
 const request = Promise.promisifyAll(require('request'));
 const uuid = require('node-uuid');
+const MachineService = require('../services/MachineService');
```

Comment:
Probably useless require.
MachineService is not included like that elsewhere in the repo.